code/modules/fishing/fish/fish_traits.dm

```diff
@@ -322,6 +325,15 @@ GLOBAL_LIST_INIT(spontaneous_fish_traits, populate_spontaneous_fish_traits())
 	stench.temperature = mob.bodytemperature
 	our_turf.assume_air(stench)
 
+/datum/fish_trait/emulsijack/psychic
+	name = "Psychic Aura"
+	catalog_description = "This fish emits an almost unblockable psychic aura that assaults minds, slowly killing all nearby fish and making humanoids have a bad time."
```

Comment:
It seems to do nothing to humanoids despite the description, or am I wrong?

Reply:
it was meant to have a negative moodlet but the system for that is awful and horible

Reply:
also this is in the wrong pr

Reply:
So, are you going to remove it, or change the description?